tubular/scripts/frontend_utils.py

```diff
@@ -106,14 +106,8 @@ def install_requirements_npm_aliases(self):
         """ Install NPM alias requirements for app to build """
         npm_aliases = self.get_npm_aliases_config()
         if npm_aliases:
-            # Install and pin NPM to latest npm@8 version
-            proc = subprocess.Popen(['npm install npm@8'], cwd=self.app_name, shell=True)
-            return_code = proc.wait()
-            if return_code != 0:
-                self.FAIL(1, 'Could not run `npm install npm@8` for app {}.'.format(self.app_name))
-
             aliased_installs = ' '.join(['{}@{}'.format(k, v) for k, v in npm_aliases.items()])
-            # Use the locally installed npm at ./node_modules/.bin/npm
+            # Use the locally installed npm at ./node_modules/.bin/npm to install aliases
```

Comment:
[question] Does this need to reference `./node_modules/.bin/npm`, or could this code path rely on just `npm install` similar to the other existing examples of `npm install`?

```python
install_aliases_proc = subprocess.Popen(
    ['npm install {}'.format(aliased_installs)],
    cwd=self.app_name,
    shell=True,
)
```

Reply:
Code path could rely on just `npm install`, I have updated it

Reply:
Great, thanks. With this updated, I don't think we need the `Use the locally installed npm at ./node_modules/.bin/npm to install aliases` comment anymore now that we're doing a standard `npm install`.